scripts/backend_tests.py

```diff
@@ -218,18 +218,18 @@ def _convert_to_test_target(path):
     for root in os.listdir(base_path):
         if any([s in root for s in ['.git', 'third_party', 'core/tests']]):
             continue
-        if root.endswith('_test.py'):
+        if root.endswith('topic_editor_test.py'):
```

Comment:
Hm. What's going on here?

Reply:
Oh, I wrote this to test the controller. Will remove it.

Reply:
Done.